js/damage.js

```diff
@@ -854,6 +856,21 @@ function checkDownload(source, target) {
 		}
 	}
 }
+
+function checkSeedBoost(holder, field) {
```

Comment:
wtf is holder? attacker works?

Reply:
If you paid attention to the Pokémon passed as the argument for the function upon calling it, you will see `p1` and `p2`, not `attacker` or `defender`. Because you might just as well equip both Pokémon with an Electric Seed on an Electric Terrain, for purely arbitrary reasons.

Reply:
this still doesn't answer my question, p1 is the same as attacker. wtf is holder.

Reply:
Uh.... holder points at the Pokémon that holds the item?

Reply:
no I mean where else is holder used, why use "holder" when attacker/p1 or defender/p2 means the same thing?